fw/src/consts.rs

```diff
@@ -23,8 +23,8 @@ pub fn app_flags() -> AppFlags {
     f
 }
 
-/// Application timeout (exit after no user input)
-pub const TIMEOUT_S: u32 = 5 * 60;
+/// Application timeout (lock after no user input)
+pub const TIMEOUT_S: u32 = 5;
```

Comment:
Is this going to lock it after 5 seconds?

Reply:
yeah, it's broken atm so that's short for testing